content/applications/services/helpdesk/advanced/close_tickets.rst

```diff
@@ -68,17 +67,20 @@ If only certain stages should be used to track days of inactivity, they can be a
    - `Customer Feedback`
    - `Closed`
 
-   Tickets can linger in the :guilabel:`Customer Feedback stage`, because once an issue is solved,
+   Tickets may linger in the :guilabel:`Customer Feedback stage`, because once an issue is solved,
    customers may not respond immediately. At that point, the tickets can be closed automatically.
```

Comment:
```suggestion
   Tickets may linger in the :guilabel:`Customer Feedback` stage, because once an issue is solved,
   customers may not respond immediately. At that point, the tickets can be closed automatically.
```